packages/wdio-browserstack-service/src/launcher.js

```diff
@@ -71,7 +71,7 @@ export default class BrowserstackLauncherService {
             return
         }
 
-        if (this.options.browserstackLocalForcedStop) {
+        if (this.options.forcedStop) {
```

Comment:
same for this one?

Reply:
yes, it is a breaking change and therefor will only land in v6

Reply:
same remark as the previous